experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -1,65 +1,77 @@
-import os
+import logging
 from pathlib import Path
 
 import yaml
 from packaging import version as version_packaging
-from rust_fml import FmlClient
+from rust_fml import FmlClient, FmlFeatureInspector
 
 from experimenter.settings import BASE_DIR
 
+logger = logging.getLogger()
+
 
 class NimbusFmlLoader:
-    BASE_PATH = Path(BASE_DIR) / "features" / "manifests" / "apps.yaml"
+    BASE_PATH = Path(BASE_DIR, "features", "manifests", "apps.yaml")
+    MANIFEST_PATH = Path(BASE_DIR, "features", "manifests")
 
     def __init__(self, application: str, channel: str, file_location=BASE_PATH):
         self.application: str = application
         self.channel: str = channel
-        self.application_data = self.get_application_data(application, file_location)
-
-    def get_fml_clients(self, versions: list[str]) -> list[FmlClient]:
-        if not self.application_data:
-            return None
-        refs = self.get_version_refs(versions)
-        file_path = self.get_file_path()
-        return [self.create_client(file_path, self.channel, r) for r in refs]
+        self.application_data = self._get_application_data(application, file_location)
 
     @staticmethod
-    def get_application_data(application_name, file_location=BASE_PATH):
-        if os.path.exists(file_location):
+    def _get_application_data(application_name, file_location=BASE_PATH):
+        """Fetch app data from local apps.yaml file to find application names, github
+        repos, fml paths within the repos, and release versions.
+        """
+        if Path.exists(file_location):
             with open(file_location) as application_yaml_file:
                 file = yaml.safe_load(application_yaml_file.read())
                 if application_name in file:
                     return file[application_name]
                 else:
                     return None
 
-    def get_file_path(self):
+    def _get_local_file_path(self):
+        """Get path to release feature manifest from experimenter (local)."""
+        path = Path(self.MANIFEST_PATH, f"{self.application}", "release.fml.yaml")
```

Comment:
The second Arg doesn't need to be an fstring. You can just reference the variable there.